modules/core/src/shaderlib/lighting/lighting.js

```diff
@@ -58,25 +59,33 @@ function getUniforms(opts = INITIAL_MODULE_OPTIONS) {
 
     lightsPosition = DEFAULT_LIGHTS_POSITION,
     lightsStrength = DEFAULT_LIGHTS_STRENGTH,
-    coordinateSystem = COORDINATE_SYSTEM.LNGLAT,
-    coordinateOrigin = DEFAULT_COORDINATE_ORIGIN,
+    coordinateSystem: fromCoordinateSystem = COORDINATE_SYSTEM.LNGLAT,
+    coordinateOrigin: fromCoordinateOrigin = DEFAULT_COORDINATE_ORIGIN,
```

Comment:
What is this syntax?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Assigning_to_new_variables_names_and_providing_default_values